docs/src/modules/components/MuiProductSelector.tsx

```diff
@@ -148,14 +148,14 @@ const coreProducts = [
   {
     id: 'joy-ui',
     name: 'Joy UI',
-    description: 'Beautiful foudational components.',
+    description: 'Delightful modern components.',
```

Comment:
I wonder if this wouldn't be closer to the product vision, so clearer. Thoughts?

```suggestion
    description: 'Material UI with a modern look.',
```

Reply:
I think that mentioning Material UI there could be confusing, as users might expect the same API with a different look. For now, I believe the current approach works better, since we don't have a long-term vision for Joy UI yet. Therefore, I wouldn't position it closer to Material UI—who knows, in the future it might become something like 'Base UI with copy-and-paste styles.'  🤷

Reply:
The upside with referencing Material UI could be:

- it will make it clear why there are two libraries that seems to do the same thing
- it will speak more directly to the people use to Material UI (from the perspective of my friend who uses Joy UI, both are more or less equivalent in terms of API / learning curve)